pyproject.toml

```diff
@@ -11,10 +11,10 @@ authors = [
 ]
 description = "Easily download, build, install, upgrade, and uninstall Python packages"
 readme = "README.rst"
+license = "MIT"
```

Comment:
This is a bit complicated.

Setuptools has many vendored dependencies (which may vary with time), so the correct license expression is not that simple. Additionally, I don't know if it is going to be trivial to derive a correct expression automatically from the code base at each release (e.g. what happens if a maintainer change a vendored dependency? how can we guarantee the license expression is up-to-date?)...

Would it make more sense to simply omit the license expression and rely on `license-files` or an automatic scan of the relevant directories?

Reply:
Let me omit it for now. I mainly used `MIT` here as that matches the existing classifier `License :: OSI Approved :: MIT License` but I'm aware of the "vendor" dilemma. This is something which can be discussed at a later point.

**IMNAL**: During the discussion on the Python packaging docs I made the argument that vendoring, if done correctly, usually doesn't change the project license as long as all vendored dependencies are included and licenses correctly.

https://github.com/pypa/packaging.python.org/pull/1662#discussion_r1880319370
https://discuss.python.org/t/pep-639-improving-license-clarity-with-better-package-metadata/2154/39